### PR TITLE
Fetch by doi

### PR DESCRIPTION
**Pull request by acabunoc**
This adds the evidenceUrl to the query -- now we can fetch by doi!

### CODE REVIEW

File: src/badgeService.js [hunk 1]
```diff
@@ -57,18 +57,22 @@ module.exports = function (apiClient, config) {
...
       } else {
-        var filtered;
-        apiClient.getBadgeInstances({
-          system: system,
-          badge: badge
-        }, clientCallback);
+        context.badge = badge || '*';
+      }
+      if (evidenceUrl) {
+        options.paginate = {
```
Comment (acabunoc) [inline src/badgeService.js]: This magically adds `?evidenceUrl=[EVIDENCE_URL]` when querying badgekit-api.

Hack-y. I'm sorry.

**Comment by acabunoc** [comment]
Fixes #10, along with https://github.com/acabunoc/badgekit-api/commit/2a40a78f8204fa929e41479de32eacd69b314ad5

**Comment by acabunoc** [comment]
@alicoding - can you r? It's the stuff we talked about yesterday, adding evidence (aka DOI) to the query.

**Comment by alicoding** [comment]
R+

**Comment by acabunoc** [comment]
thanks Ali!

**Pull request merged**